psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -425,19 +425,13 @@ void MintsHelper::one_body_ao_computer(std::vector<std::shared_ptr<OneBodyAOInt>
     std::shared_ptr<BasisSet> bs1 = ints[0]->basis1();
     std::shared_ptr<BasisSet> bs2 = ints[0]->basis2();
 
-    // Limit to the number of incoming onebody ints
-    size_t nthread = nthread_;
-    if (nthread > ints.size()) {
-        nthread = ints.size();
-    }
-
```

Comment:
Why is this removed?

Reply:
Each vector is already formed using nthread_ as the max length. Checking seemed superfluous and unnecessary.

```
std::vector<std::shared_ptr<OneBodyAOInt>> ints_vec;
    for (size_t i = 0; i < nthread_; i++) {
        ints_vec.push_back(std::shared_ptr<OneBodyAOInt>(integral_->ao_overlap()));
    }
```